2021-04-25 17:24:51,711 INFO    MainThread:53580 [wandb_setup.py:_flush():69] setting env: {}
2021-04-25 17:24:51,712 INFO    MainThread:53580 [wandb_setup.py:_flush():69] setting login settings: {}
2021-04-25 17:24:51,712 INFO    MainThread:53580 [wandb_init.py:_log_setup():336] Logging user logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_172451-tygba858/logs/debug.log
2021-04-25 17:24:51,712 INFO    MainThread:53580 [wandb_init.py:_log_setup():337] Logging internal logs to /md0/home/btilmon/projects/learncfa_pytorch/wandb/run-20210425_172451-tygba858/logs/debug-internal.log
2021-04-25 17:24:51,713 INFO    MainThread:53580 [wandb_init.py:init():369] calling init triggers
2021-04-25 17:24:51,713 INFO    MainThread:53580 [wandb_init.py:init():376] wandb.init called with sweep_config: {}
config: {}
2021-04-25 17:24:51,713 INFO    MainThread:53580 [wandb_init.py:init():418] starting backend
2021-04-25 17:24:51,713 INFO    MainThread:53580 [backend.py:_multiprocessing_setup():71] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-04-25 17:24:51,731 INFO    MainThread:53580 [backend.py:ensure_launched():123] starting backend process...
2021-04-25 17:24:51,739 INFO    MainThread:53580 [backend.py:ensure_launched():128] started backend process with pid: 53593
2021-04-25 17:24:51,741 INFO    MainThread:53580 [wandb_init.py:init():423] backend started and connected
2021-04-25 17:24:51,744 INFO    MainThread:53580 [wandb_init.py:init():463] updated telemetry
2021-04-25 17:24:51,745 INFO    MainThread:53580 [wandb_init.py:init():482] communicating current version
2021-04-25 17:24:52,475 INFO    MainThread:53580 [wandb_init.py:init():487] got version response upgrade_message: "wandb version 0.10.27 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-04-25 17:24:52,475 INFO    MainThread:53580 [wandb_init.py:init():495] communicating run to backend with 30 second timeout
2021-04-25 17:24:52,653 INFO    MainThread:53580 [wandb_init.py:init():520] starting run threads in backend
2021-04-25 17:24:52,716 INFO    MainThread:53580 [wandb_run.py:_console_start():1524] atexit reg
2021-04-25 17:24:52,717 INFO    MainThread:53580 [wandb_run.py:_redirect():1371] redirect: SettingsConsole.REDIRECT
2021-04-25 17:24:52,717 INFO    MainThread:53580 [wandb_run.py:_redirect():1376] Redirecting console.
2021-04-25 17:24:52,720 INFO    MainThread:53580 [wandb_run.py:_redirect():1438] Redirects installed.
2021-04-25 17:24:52,721 INFO    MainThread:53580 [wandb_init.py:init():544] run started, returning control to user process
2021-04-25 17:25:12,511 INFO    MainThread:53580 [wandb_run.py:_atexit_cleanup():1494] got exitcode: 255
2021-04-25 17:25:12,511 INFO    MainThread:53580 [wandb_run.py:_restore():1466] restore
2021-04-25 17:25:14,682 INFO    MainThread:53580 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 17:25:15,059 INFO    MainThread:53580 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 758
}

2021-04-25 17:25:15,676 INFO    MainThread:53580 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 2
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 760
}

2021-04-25 17:25:15,862 INFO    MainThread:53580 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 758
  total_bytes: 1966
}

2021-04-25 17:25:15,964 INFO    MainThread:53580 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1966
  total_bytes: 1966
}

2021-04-25 17:25:16,067 INFO    MainThread:53580 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1966
  total_bytes: 1966
}

2021-04-25 17:25:16,169 INFO    MainThread:53580 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1966
  total_bytes: 1966
}

2021-04-25 17:25:16,271 INFO    MainThread:53580 [wandb_run.py:_wait_for_finish():1616] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1966
  total_bytes: 1966
}

2021-04-25 17:25:16,374 INFO    MainThread:53580 [wandb_run.py:_wait_for_finish():1616] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 1966
  total_bytes: 1966
}

2021-04-25 17:25:17,501 INFO    MainThread:53580 [wandb_run.py:_show_files():1838] logging synced files
